pkg/provisioning/cbnt/validation_help.go

```diff
@@ -0,0 +1,52 @@
+package cbnt
+
+import (
+	"crypto"
+	"crypto/ecdsa"
+	"crypto/rsa"
+	"fmt"
+
+	"github.com/9elements/converged-security-suite/v2/pkg/intel/metadata/manifest"
+	"github.com/tjfoc/gmsm/sm2"
+)
+
+func validateSignature(s *manifest.Signature, unsigneddata []byte, key crypto.PublicKey) error {
+	switch s.SigScheme {
+	case manifest.AlgRSAPSS:
```

Comment:
nit: Both RSA cases could be grouped together?

Reply:
done